server/handler/pull_request_review.go

```diff
@@ -45,6 +45,8 @@ func (h *PullRequestReview) Handle(ctx context.Context, eventType, deliveryID st
 	installationID := githubapp.GetInstallationIDFromEvent(&event)
 	ctx, logger := githubapp.PreparePRContext(ctx, installationID, repo, number)
 
+	logger.Debug().Msgf("received pull_request_review %s event", event.GetAction())
```

Comment:
The `action` field is always going to be submitted in this case, so it might not be all that useful. However, keeping the value ensures the messages match those for other event types.

Would it be helpful to include the user information here?